hack/verify-generated-swagger-docs.sh

```diff
@@ -55,7 +55,7 @@ cp -a "${DIFFROOT}"/* "${TMP_DIFFROOT}/"
 "${KUBE_ROOT}/hack/update-generated-swagger-docs.sh"
 echo "diffing ${DIFFROOT} against freshly generated swagger type documentation"
 ret=0
-diff -Naupr -I 'Auto generated by' "${DIFFROOT}" "${TMP_DIFFROOT}" || ret=$?
+diff --exclude=".import-restrictions" -Naupr -I 'Auto generated by' "${DIFFROOT}" "${TMP_DIFFROOT}" || ret=$?
```

Comment:
If you regenerate the docs completely, does the file get wiped out?

Reply:
y, running `hack/update-generated-swagger-docs.sh` seems to get rid of the file

Reply:
so `--exclude` may not be a great idea :)

Reply:
yeah, that's my thoughts. not sure how to handle the import-restrictions file then =\

Reply:
Hmm, sorry i checked again, the generated files in _tmp/pkg does not have the file, We don't wipe the existing file from disk. So this change is still legit

```
Generating swagger type docs for settings.k8s.io/v1alpha1 at vendor/k8s.io/api/settings/v1alpha1
Generating swagger type docs for storage.k8s.io/v1beta1 at vendor/k8s.io/api/storage/v1beta1
Generating swagger type docs for storage.k8s.io/v1 at vendor/k8s.io/api/storage/v1
diffing /Users/dims/go/src/k8s.io/kubernetes/pkg against freshly generated swagger type documentation
diff -Naupr -I 'Auto generated by' /Users/dims/go/src/k8s.io/kubernetes/pkg/.import-restrictions /Users/dims/go/src/k8s.io/kubernetes/_tmp/pkg/.import-restrictions
--- /Users/dims/go/src/k8s.io/kubernetes/pkg/.import-restrictions	2017-10-07 21:36:12.000000000 -0400
+++ /Users/dims/go/src/k8s.io/kubernetes/_tmp/pkg/.import-restrictions	1969-12-31 19:00:00.000000000 -0500
@@ -1,13 +0,0 @@
-{
-	"Rules": [
-	  {
-		"SelectorRegexp": "k8s[.]io/kubernetes",
-		"AllowedPrefixes": [
-		    ""
-		],
-		"ForbiddenPrefixes": [
-			"k8s.io/kubernetes/cmd"
-		]
-	  }
-	]
-}
/Users/dims/go/src/k8s.io/kubernetes/pkg is out of date. Please run hack/update-generated-swagger-docs.sh
```